SnapSpire/Features/HomeListView.swift

```diff
@@ -0,0 +1,77 @@
+import SwiftUI
+
+struct HomeListView: View {
+    let columns = [GridItem(.flexible())]
+    let user = User.sampleData
+
+    var body: some View {
+        NavigationStack {
+            ScrollView {
+                LazyVGrid(columns: columns) {
```

Comment:
After thinking about it, maybe la lazy grid is too much, let's just use a VStack